addons/sourcemod/scripting/vsh/bosses/boss_horsemann.sp

```diff
@@ -136,8 +157,21 @@ methodmap CHorsemann < SaxtonHaleBase
 	
 	public Action OnSoundPlayed(int clients[MAXPLAYERS], int &numClients, char sample[PLATFORM_MAX_PATH], int &channel, float &volume, int &level, int &pitch, int &flags, char soundEntry[PLATFORM_MAX_PATH], int &seed)
 	{
-		if (strncmp(sample, "vo/", 3) == 0)//possibly look replacing into one of HHH sound?
+		if (strncmp(sample, "vo/", 3) == 0)
+		{
+			if (StrContains(sample, "vo/halloween_boss/", false) != -1)
+				return Plugin_Continue;
+			
+			EmitSoundToAll(g_strHorsemannVoice[GetRandomInt(0, sizeof(g_strHorsemannVoice) - 1)], this.iClient, SNDCHAN_VOICE);
+			return Plugin_Handled;
+		}
+			
+		if (StrContains(sample, "player/footsteps/", false) != -1)
+		{
+			EmitSoundToAll(g_strHorsemannFootsteps[GetRandomInt(0, sizeof(g_strHorsemannFootsteps) - 1)], this.iClient, _, _, _, 0.4, GetRandomInt(90, 100));
 			return Plugin_Handled;
```

Comment:
ditto

Reply:
All changes that were requested were done in a way or another except for this one, seems like footsteps don't respect `return Plugin_Changed` (or even `return Plugin_Handled` for that matter)